App/src/main/java/cc/softwarefactory/lokki/android/MainActivity.java

```diff
@@ -277,22 +280,15 @@ public boolean onOptionsItemSelected(MenuItem item) {
 
     private void toggleVisibility() {
 
-        MainApplication.visible = !MainApplication.visible;
-        int visibility_mode = MainApplication.visible ? 0 : 1;
-        PreferenceUtils.setValue(this, PreferenceUtils.KEY_SETTING_VISIBILITY, String.valueOf(visibility_mode));
+        Utils.setVisibility(!MainApplication.visible, MainActivity.this); // oh the side effects...
```

Comment:
What side effects? I don't think it is a good idea to leave vague comments like this without explaining them or taking them out.

Also you might want to use the application context instead of just the `MainActivity.this` context.